enterprise/server/scheduling/priority_task_scheduler/priority_task_scheduler.go

```diff
@@ -463,3 +475,13 @@ func (q *PriorityTaskScheduler) GetQueuedTaskReservations() []*scpb.EnqueueTaskR
 	defer q.mu.Unlock()
 	return q.q.GetAll()
 }
+
+// Formats a large number as groups of 3 digits separated by underscores.
```

Comment:
Elsewhere we use the following which gives comma separated numbers:
```
	p := message.NewPrinter(language.English)
    return p.Sprint(number)`
```

Reply:
ah cool, TIL. Done!